File: array_problems/flattenArray/flattenArray.py
import json

with open('../array.json') as file:
    array = json.load(file)
class ArrayProblems:
    def flatten_arr(self, arr):
        flatten_arr = []
        for i in range(len(arr)):
            if isinstance(arr[i], list):
                flatten_arr.extend(self.flatten_arr(arr[i]))
            else:
                flatten_arr.append(arr[i])


        return flatten_arr


array_problems = ArrayProblems()
print(array_problems.flatten_arr(array))

File: array_problems/flattenArray/flattenArray.php
<?php
$json_data = file_get_contents('../array.json');
$array = json_decode($json_data, true);
class ArrayProblems
{
    public function flatten_array($arr)
    {
        $flatten_arr = [];
        for($i = 0; $i < count($arr); $i++) {
            if (is_array($arr[$i])) {
                $flatten_arr = array_merge($flatten_arr, $this->flatten_array($arr[$i]));
            } else {
                array_push($flatten_arr, $arr[$i]);
            }
        }
        return $flatten_arr;
    }
}
$array_problems = new ArrayProblems();
$res = $array_problems->flatten_array($array);
print_r($res);